eng/Signing.props

```diff
@@ -0,0 +1,9 @@
+<Project>
```

Comment:
Do we really need this empty file? It is only imported when it exists: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Sign.proj#L48